sampleresources.md

```diff
@@ -0,0 +1,15 @@
+#Helpful sample resorces for Microsoft ASP.NET WebHooks
+Please see the initial [documentation](http://docs.asp.net/projects/aspnetwebhooks/en/latest/) for details.
+
+Following(s) are helpful resources from our contributors: 
+
+##Samples
+- [GitHub](https://github.com/aspnet/WebHooks/tree/master/samples/#)
```

Comment:
Ah, I see. What is the `GitHub` link then? Maybe just call it "All samples"? (I interpreted it to mean "A sample for using GitHub web hooks".)